webserver/templates/datasets/eval-job.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+
+{% block title %}Dataset "{{ dataset['name'] }}" by {{ author.musicbrainz_id }} - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <div id="evaluation-results">
+
+    {% from 'datasets/macros.html' import print_header %}
+    {{ print_header(dataset, author, current_user, 'evaluation') }}
+
+    {% if job['status'] == 'pending' %}
+      <div id="status" class="alert alert-info">
+        This dataset has not been evaluated yet.
+      </div>
+
+    {% elif job['status'] == 'running' %}
+      <div id="status" class="alert alert-info">
+        This dataset is being evaluated right now. Results should be available soon.
+      </div>
+
+    {% elif job['status'] == 'failed' %}
+      <div id="status" class="alert alert-danger">
+        Evaluation of this dataset has failed!
```

Comment:
That's a good idea. If it hasn't been updated since it failed, don't let them change it.

However, this doesn't solve the issue of us having a gaia error which is something we need to fix. Do you have any ideas on a way to report errors to us so that we can fix them, and then let us re-queue someone's dataset?  Perhaps we could use sentry.